src/backend/auth.api.ts

```diff
@@ -190,7 +190,15 @@ const getCurrentUser = async () => {
  * @returns {Object} returns session
  */
 const logoutUser = async () => {
-  return await account.deleteSession("current");
+  try {
+    // Use the deleteSession method without specifying a session ID
+    const deletedSession = await account.deleteSession();
```

Comment:
The session is clearing from the appwrite but not clearing from the cookies. Try to implement it in the redux where we are updating the cookies